bigtop-packages/src/charm/spark/layer-spark/tests/01-basic-deployment.py

```diff
@@ -22,14 +22,13 @@ class TestDeploy(unittest.TestCase):
     """
     Trivial deployment test for Apache Spark.
     """
-    def setUp(self):
+
+    def test_deploy(self):
```

Comment:
as discussed earlier today, would you mind making this a class method?  something like:

https://github.com/juju-solutions/bigtop/blob/PIG-BIGTOP-2482/bigtop-packages/src/charm/pig/layer-pig/tests/01-deploy.py#L27